evennia/contrib/game_systems/storage/storage.py

```diff
@@ -0,0 +1,180 @@
+from evennia import CmdSet
+from evennia.utils import list_to_string
+from evennia.utils.search import search_object_by_tag
+from evennia.commands.default.muxcommand import MuxCommand
+
+
+class StorageCommand(MuxCommand):
+    """
+    Shared functionality for storage-related commands
+    """
+
+    def at_pre_cmd(self):
+        """
+        Check if the current location is tagged as a storage location
+        Every stored object is tagged on storage, and untagged on retrieval
+        """
+        success = super().at_pre_cmd()
```

Comment:
A nitpick: If `at_pre_cmd` returns `False`, the command execution should abort. By default the parent doesn't do anything, but if a dev changes the command base, I think the parent's result should be respected and return/abort already here.

Reply:
I think this is actually the opposite (https://www.evennia.com/docs/latest/_modules/evennia/commands/command.html#Command.at_pre_cmd) but point stands anyway, changed to return True if the super() returns True before checking further.